src/components/leaflet/Leaflet_map.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Let's agree on file naming. Please don't use underscores. PascalCase always for components and camelCase for helpers.